pkg/network/node/node.go

```diff
@@ -366,6 +367,19 @@ func (node *OsdnNode) Start() error {
 	}
 
 	klog.V(2).Infof("openshift-sdn network plugin ready")
+
+	klog.V(2).Infof("Checking default interface MTU")
+	mtu, err := GetDefaultMTU()
+
+	if err != nil {
+		return err
+	}
+
+	if mtu <= c.MTU {
```

Comment:
You need to account for the 50-byte vxlan overhead here. In other words, a 1500 mtu means that c.MTU can be 1450